packages/lib-utils/src/app/redux/index.ts

```diff
@@ -0,0 +1,43 @@
+import { consoleLogger } from '@monorepo/common';
+import * as React from 'react';
+import { useStore } from 'react-redux';
+import type { Store } from 'redux';
+import { applyMiddleware, combineReducers, createStore, compose } from 'redux';
+import thunk from 'redux-thunk';
+import { setReduxStore, getReduxStore } from '../../config';
+import type { SDKStoreState } from '../../types/redux';
+import { SDKReducers } from './reducers';
+
+/**
+ * `useReduxStore` will provide the store instance if present or else create one along with info if the context was present.
+ *
+ * @example
+ * ```ts
+ * function Component () {
+ *   const {store, storeContextPresent} = useReduxStore()
+ *   return ...
+ * }
+ * ```
+ */
+export const useReduxStore = (): { store: Store<unknown>; storeContextPresent: boolean } => {
+  const storeContext = useStore();
```

Comment:
```suggestion
  const storeInUse = useStore();
```

Reply:
@vojtechszocs proposing we keep `storeContext` for now - as the state variable `storeContextPresent` would also need to be renamed and `storeInUsePresent` sounds a bit odd to me.